workbench/_api/routes/lens.py

```
@@ -120,43 +111,16 @@ def process_line_results(
-        return {"job_id": result}
-
-    return {"data": process_line_results(result, req, state)}
-
-
-@router.post("/results-line/{job_id}", response_model=LensLineResponse)
-async def collect_line(
-    job_id: str,
+@router.post("/run-line")
+async def run_line(
     req: LensLineRequest,
     state: AppState = Depends(get_state),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)